src/utils.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable import/prefer-default-export */
+const todos = JSON.parse(localStorage.getItem('todos')) || [];
+export const addTodo = (description) => {
+  const index = todos && todos.length > 0 ? todos.length + 1 : 1;
+  const todo = { description, index, completed: false };
+  todos.push(todo);
+  localStorage.setItem('todos', JSON.stringify(todos));
```

Comment:
Hi @benshidanny11,

Great Work implementing the CRUD function, I like the fact that you follow best practices for HTML, CSS, and JavaScript.

In this code snippet, you repeated the statement that stores your objects in the local storage and the `window.location.reload()` function. Part of JavaScript practices is to try and make your code as **_DRY_** as possible.
So I will suggest that you put it in a function and call it in when you need it.

I hope to collaborate with you in the future. Happy Coding 👍